pkg/controller/mobiledeveloperconsole/mobiledeveloperconsole_controller.go

```diff
@@ -195,6 +226,19 @@ func (r *ReconcileMobileDeveloperConsole) Reconcile(request reconcile.Request) (
 	}
 	//#endregion
 
+	//#region DeleteOldRoleBinding
+	// TODO: This can be removed after a release or two
```

Comment:
@grdryn as there might be some additional upgrade logic in here wdyt about adding the release that the upgrade is relevant for to this comment so that when we see the TODO later we know which release would be two after?

Reply:
We're not really sure what version it would be removable in yet. It would at least be a release or two after an RHMI release. Would versions in JIRA be a good way to track this stuff?

Reply:
Going to merge this now to unblock @jjaferson, we can add in anything we want after :+1: